apps/io-wallet-user-func/src/email-provider.ts

```diff
@@ -0,0 +1,55 @@
+import emailTemplate from "@/templates/wallet-instance-activation/index.html";
+import {
+  getMailerTransporter,
+  sendMail,
+} from "@pagopa/io-functions-commons/dist/src/mailer";
+import * as RTE from "fp-ts/lib/ReaderTaskEither";
+import * as TE from "fp-ts/lib/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+
+import { MailConfig } from "./app/config";
+
+export const sendEmail: (
+  fiscalCode: string,
+) => RTE.ReaderTaskEither<{ mail: MailConfig }, Error, void> =
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  (fiscalCode) => (configs) =>
+    pipe(
+      TE.tryCatch(
+        async () => {
+          if (!configs.mail.enabled) {
+            return;
+          }
+          const userEmail = ""; // to do - get the email by fiscalCode
+
+          await sendMail(
+            getMailerTransporter({
+              MAIL_FROM: configs.mail.mailhogHost?.length
+                ? configs.mail.mailSender
+                : userEmail,
+              MAIL_TRANSPORTS: undefined,
+              MAILHOG_HOSTNAME: configs.mail.mailhogHost?.length
+                ? configs.mail.mailhogHost
+                : undefined,
+              MAILUP_SECRET: configs.mail.mailupSecret?.length
+                ? configs.mail.mailupSecret
+                : undefined,
+              MAILUP_USERNAME: configs.mail.mailupUsername?.length
+                ? configs.mail.mailupUsername
+                : undefined,
```

Comment:
are you sure these undefined values ​​are needed?

Reply:
I think so! I've tried a few things and it seems to be expecting exactly `undefined`.